evm/src/cpu/kernel/interpreter.rs

```diff
@@ -1129,14 +1165,20 @@ impl<'a> Interpreter<'a> {
         Ok(())
     }
 
-    fn jump_to(&mut self, offset: usize) -> anyhow::Result<(), ProgramError> {
-        // The JUMPDEST rule is not enforced in kernel mode.
-        if !self.is_kernel() && self.jumpdests.binary_search(&offset).is_err() {
```

Comment:
It seems that `self.jumpdest` isn't needed anymore?

Reply:
Same for `find_jumpdests()`